storage/innobase/xtrabackup/test/t/history_on_server.sh

```diff
@@ -85,6 +85,15 @@ do
     check_for_value "$column" "N"
 done
 
+get_one_value "lock_time"
+lock_time_without_lock=$val
+
+if [ $val -lt 0 ];
```

Comment:
Less than or equal to 0 is also wrong. Please update check

Reply:
below are the results of manual testing with lock options. 

mysql> select * from PERCONA_SCHEMA.xtrabackup_history\G                                 *************************** 1. row ***************************
            uuid: 6e31a43a-3dbb-11ea-9398-002590d40aec
            name: NULL
       tool_name: xtrabackup
    tool_command: --target-dir=/home/rahul.malik/MySQL/data/xo7 --core-file --user=root --socket /tmp/socket_1320.sock --keyring-file-data=/home/rahul.malik/MySQL/data/o7/key.key --xtrabackup-plugin-dir=/home/rahul.malik/MySQL/src/x7/bld/storage/innobase/xtrabackup/src/keyring --history --backup
    tool_version: 2.4.18
ibbackup_version: 2.4.18
  server_version: 5.7.29-debug
      start_time: 2020-01-23 08:34:02
        end_time: 2020-01-23 08:36:20
       lock_time: 136
      binlog_pos: NULL
 innodb_from_lsn: 0
   innodb_to_lsn: 172601168
         partial: N
     incremental: N
          format: file
         compact: N
      compressed: N
       encrypted: N
1 row in set (0.00 sec)

Reply:
ideally mtr testcase would be nice. but since it cannot be tested easily ie getting non zero lock time is not easy, manual result proof is fine.